src/Microsoft.Health.Fhir.TemplateManagement/OCIFileManager.cs

```diff
@@ -50,7 +51,8 @@ public async Task<ImageInfo> PullOCIImageAsync()
         /// <param name="manifest">Manifest of the image.</param>
         public void UnpackOCIImage(ManifestWrapper manifest)
         {
-            var rawLayers = _overlayFS.ReadImageLayers(manifest);
+            var rawLayers = _overlayFS.ReadImageLayers();
+            var sortedLayers = _overlayOperator.Sort(rawLayers, manifest);
```

Comment:
Update the rawLayers to sortedLayers.

Reply:
Thanks！ Fixed it.

Reply:
it seems the overlayFS already know if it's base layer or not, then can you move the sort logic into overlayfs?